### PR TITLE
[MLDB-1544] added macro to invoke logger + a logger test

### PR DESCRIPTION
**Pull request by guyd**

### CODE REVIEW

File: plugins/classifier.cc [hunk 1]
```diff
@@ -106,7 +106,7 @@ ClassifierConfigDescription()
              0.5);
     addField("mode", &ClassifierConfig::mode,
              "Mode of classifier.  Controls how the label is interpreted and "
-             "what is the output of the classifier.", CM_BOOLEAN);    
+             "what is the output of the classifier.", CM_BOOLEAN);
```
Comment (guyd) [inline plugins/classifier.cc]: Sorry for the noise.  I decided to start fighting trailing whitespaces!

**Comment by steadtler** [comment]
Im not sure about the motivation but it looks ok +1

**Comment by guyd** [comment]
The main motivation is to be able to leave log message uncommented without fear of having them impact the performance of MLDB.

**Comment by steadtler** [comment]
Ah the Jira phrased it strangely. Thanks.

**Pull request merged**